{"id":27654,"className":"javax.xml.crypto.dsig.XMLSignatureException","methodName":"getCause","signature":"\u003cjavax.xml.crypto.dsig.XMLSignatureException: java.lang.Throwable getCause()\u003e","javadoc":"/** \n * Returns the cause of this \u003ccode\u003eXMLSignatureException\u003c/code\u003e or \u003ccode\u003enull\u003c/code\u003e if the cause is nonexistent or unknown.  (The cause is the throwable that caused this \u003ccode\u003eXMLSignatureException\u003c/code\u003e to get thrown.)\n * @return the cause of this \u003ccode\u003eXMLSignatureException\u003c/code\u003e or\u003ccode\u003enull\u003c/code\u003e if the cause is nonexistent or unknown.\n */\n","tf":{"freq":{"thrown":1,"except":3,"caus":7,"nonexist":2,"lsignatur":3,"throwabl":1,"unknown":2},"maxFreq":7,"totalTerms":19},"loc":3}